docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>I think I'm being too picky here, but I think we should add more emphasis to packages like brms and rstanarm instead of nlme (I don't see it being mentioned that much in the R world). I would say "R packages like lme4 or brms" but it's completely fine if you don't want to change it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>